.github/workflows/update-spincrystals.yml

```diff
@@ -0,0 +1,52 @@
+name: Spincrystals Update
+
+on:
+  schedule:
+    # Runs at 1:00 AM every Thursday every 2 weeks
+    - cron: '0 1 * * 4/2'
```

Comment:
You can run this every day ^^

Reply:
Does it need to be ran every day though? These aren't updated very often (every 2 patches or so)

Reply:
Yeah but if the job misses the changes it has to wait 2 weeks for data to be updated